superset-frontend/src/chart/ChartErrorMessage.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import {
+  useApiV1Resource,
+  useResourceTransform,
+} from 'src/common/hooks/apiResources';
+import ErrorMessageWithStackTrace from 'src/components/ErrorMessage/ErrorMessageWithStackTrace';
+import { SupersetError } from 'src/components/ErrorMessage/types';
+import Chart from 'src/types/Chart';
+
+interface Props {
+  chartId: string;
+  error?: SupersetError;
+}
+
+function extractOwnerNames({ owners }: Chart) {
+  if (!owners) return null;
+  return owners.map(owner => `${owner.first_name} ${owner.last_name}`);
+}
+
+function useChartOwnerNames(chartId: string) {
+  const { result } = useResourceTransform(
+    useApiV1Resource<Chart>(`/api/v1/chart/${chartId}`),
```

Comment:
note: If you want you can just select the columns you want by:
`/api/v1/chart/501?q=(columns:!(owners.first_name,owners.last_name,owners.id),keys:!(none))`

Reply:
I expect if more of these requests are being made in the future, and we've changed the `useApiResource` hooks to use shared storage, it would actually be better for performance to request the entire object. That way if one hook wants owners and another hook wants creators, or name, or whatever, they can all get what they need with one request.

For now though, this is definitely a good add. Thanks for the tip!